mappings/net/minecraft/world/chunk/ChunkManager.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_51 net/minecraft/world/chunk/ChunkManager
+	METHOD method_1801 tick ()Z
+	METHOD method_1802 hasChunk (II)Z
```

Comment:
[isChunkLoaded](https://maven.fabricmc.net/docs/yarn-1.21+build.2/net/minecraft/world/chunk/ChunkManager.html#isChunkLoaded(int,int)) in Yarn